src/exchanges/binance/time/log-time-sync.ts

```diff
@@ -0,0 +1,49 @@
+import { USDMClient } from 'binance';
+
+const client = new USDMClient();
+
+/**
+ * Script to synchronize local time with Bybit's server time and log latency estimates
```

Comment:
```suggestion
 * Script to measure local time differences with Bybit's server time and log latency estimates
```

Would suggest different phrasing, because "synchronise" suggests it resolves time issues, while this just logs it and suggests a change (which is still useful ofc).